app/cdap/components/PipelineDetails/PipelineDetailsTopPanel/PipelineDetailsDetailsActions/PipelineDetailsActionsButton/index.js

```diff
@@ -31,10 +31,21 @@ import downloadFile from 'services/download-file';
 import { santizeStringForHTMLID } from 'services/helpers';
 import { deleteEditDraft } from 'components/PipelineList/DeployedPipelineView/store/ActionCreator';
 import { DiscardDraftModal } from 'components/shared/DiscardDraftModal';
+import styled from 'styled-components';
 require('./PipelineDetailsActionsButton.scss');
 
 const PREFIX = 'features.PipelineDetails.TopPanel';
 
+const StyledLi = styled.li`
+  ${({ isLatestVersion }) =>
+    !isLatestVersion &&
+    `opacity: 0.5;
+    &:hover {
+      background: white !important;
```

Comment:
whats going on with all these importants? can you make it more specific instead?